src/main/java/browserview/ChromeDriverEx.java

```diff
@@ -109,6 +109,16 @@ public WebElement findElement(By by) throws NoSuchElementException {
         throw new NoSuchElementException();
     }
 
+    public void scrollToTop() {
+        String script = "window.scrollTo(0, 0)";
+        executeScript(script);
+    }
+
+    public void scrollToBottom() {
+        String script = "window.scrollTo(0, document.body.scrollHeight)";
+        executeScript(script);
+    }
+
```

Comment:
We already had hot keys to scroll to top and bottom. So why the need to add these methods now? We should use the same methods in both places right?